packages/cli/config/index.ts

```diff
@@ -590,13 +590,13 @@ const config = convict({
 			},
 			maxFileSize: {
 				doc: 'Maximum size for each log file in MB.',
-				format: String,
+				format: Number,
 				default: '16',
 				env: 'N8N_LOG_FILE_MAXSIZE',
 			},
 			maxFileCount: {
 				doc: 'Maximum number of files to keep.',
-				format: String,
+				format: Number,
 				default: '100',
```

Comment:
default values need to be updated too since type is now Number

Reply:
Again: thanks, again :D hahaha

Reply:
😊✅